packages/asset-swapper/src/types.ts

```diff
@@ -227,34 +227,34 @@ export type SwapQuoteOrdersBreakdown = Partial<
 >;
 
 /**
- * nativeExclusivelyRFQT: if set to `true`, Swap quote will exclude Open Orderbook liquidity.
- *                        If set to `true` and `ERC20BridgeSource.Native` is part of the `excludedSources`
- *                        array in `SwapQuoteRequestOpts`, an Error will be raised.
+ * nativeExclusivelyRFQ: if set to `true`, Swap quote will exclude Open Orderbook liquidity.
+ *                       If set to `true` and `ERC20BridgeSource.Native` is part of the `excludedSources`
+ *                       array in `SwapQuoteRequestOpts`, an Error will be raised.
  */
-export interface RfqtRequestOpts {
+export interface RfqRequestOpts {
     takerAddress: string;
     txOrigin: string;
     apiKey: string;
     intentOnFilling: boolean;
     isIndicative?: boolean;
     makerEndpointMaxResponseTimeMs?: number;
-    nativeExclusivelyRFQT?: boolean;
-    altRfqtAssetOfferings?: AltRfqtMakerAssetOfferings;
+    nativeExclusivelyRFQ?: boolean;
+    altRfqtAssetOfferings?: AltRfqMakerAssetOfferings;
```

Comment:
left this field to be called `altRfqtAssetOfferings` because not sure if this will be separate for RFQT and RFQM

Reply:
no it should be the same